sdk/iothub/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/iothub/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/iothub/
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: iothub
+    ArtifactName: packages
+    Artifacts:
```

Comment:
track 1 SDKs cannot be published like the track 2 SDKs
You don't need this yaml file just like the track 1 SDK for digital twin